source/faq.txt

```diff
@@ -107,3 +107,20 @@ have more than 2 non-arbiter nodes and you must sync from a non-arbiter
 node. Use the source cluster's connection string to specify a
 :ref:`read preference <mongodb-uri>` for a non-arbiter, data-bearing
 node. 
+
+What If I see a Slow Operation Warning?
+---------------------------------------
+
+Slow operation warnings can occur during the initial sync or the application 
+of a change event when there is a slow read operation on the source cluster 
+or a slow write operation on the destination cluster.  The warning may
+indicate network congestion or resource strain on the source or destination
+cluster.
+
+Though the warning itself is harmless, slow operations can cause operation
```

Comment:
I'm wondering if we can clarify what "harmless" means here. Maybe:

While these warnings do not indicate failures themselves, slow operations can cause operation timeout errors timeout errors in ``mongosync`` and migration failures.

Reply:
Fixed